test/common/types.go

```diff
@@ -19,3 +27,11 @@ type TestCase struct {
 	Description string
 	Test        func(t *testing.T, ctx *TestingContext)
 }
+
+type apiResponse struct {
```

Comment:
If this is going to be stored in the `common` package we should probably make the name of this clearer so it's clear what it's an API response from. wdyt?

Reply:
Yes good point!